src/ontology/uberon-edit.obo

```diff
@@ -224246,6 +224246,119 @@ intersection_of: part_of UBERON:0001987 ! placenta
 relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
 property_value: dcterms-date 2022-06-27T09:30:52Z xsd:dateTime
 
+[Term]
+id: UBERON:8480009
+name: perianal space
+def: "The virtual spaces circumscribed by the mucosa lining of the anal canal and the muscular structures of the sphincter complex." [http://www.endoscopy-colon-explorer.com/anatomy-of-the-anal-region/]
+is_a: UBERON:0000464 ! anatomical space
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+relationship: part_of UBERON:0001353 ! anal region
+property_value: dcterms-date 2022-07-05T10:24:30Z xsd:dateTime
+
+[Term]
+id: UBERON:8480010
+name: subcutaneous perianal space
+def: "A space located below the dentate line. Its borders are the skin of the anal verge below and the internal sphincter above." [http://www.endoscopy-colon-explorer.com/anatomy-of-the-anal-region/]
```

Comment:
"A perianal space located..." <- we also try to use the direct parent in the def

Reply:
make this change for all the terms below

Reply:
OK thanks